scrapers/JimWeathersArchives.yml

```
@@ -0,0 +1,54 @@
+name: "Jim Weathers Archives"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jimweathersarchives.com
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - jimweathersarchives.com
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class='title_bar']/span/text()
+      URL: 
+        selector: //div[@class='rearrangeTab options_block active']/a[@class='option_button']/@href
+      Details:
+        selector: //span[@class='update_description']
+      Date:
+        selector: //div[@class='gallery_info']/div[@class='table']/div[@class='row']/div[@class='cell update_date']
+        postProcess:
+          - parseDate: 01/02/2006
+      Image:
+        selector: //meta[@name="twitter:image"]/@content
+        postProcess:
+          - replace:
+              - regex: ^(.+)\-.+
+                with: $1-4x.jpg
+      Studio:
+        Name:
+          fixed: BondageCafe
+      Tags:
+        Name:
+          selector: //span[@class='update_tags']/a
+      Performers:
+        Name:
+          selector: //div[@class='gallery_info']/span[@class='update_models']/a
+        URL: 
+          selector: //div[@class='gallery_info']/span[@class='update_models']/a/@href         
+  performerScraper:
+    performer:
+      Name:
+        selector: //span[@class='title_bar_hilite']/text()
+      Gender:
+        fixed: female     
+      Image:
+        selector: //img[@id='set-target-13']/@src0_3x

```

the `set-target-xx` is not always 13 (eg `https://www.jimweathersarchives.com/store/models/DianaKnight.html`, `https://www.jimweathersarchives.com/store/models/KendraJames.html`), i suggest switching to something like 
```suggestion
        selector: //img[contains(@class,'model_bio_thumb')]/@src0_3x
```
instead

---

Noticed that and fixed it on my previous version, but it got lost during some other testing. Thanks for catching it.